source/core/parameters.cc

```diff
@@ -1857,6 +1857,15 @@ namespace Parameters
         "ib_force",
         Patterns::FileName(),
         "The name of the file where the data on the force of each particle is stored");
+      prm.declare_entry(
+        "load particles from a file",
```

Comment:
In the documentation, the parameter is called 'load particles from file', and here 'load particles from a file'. I would use the first version since it is a parameter and it can be quite annoying to miss the 'a' on the setup of the simulation.